src/generated/linux/datapath_winuser.c.clog.h

```diff
@@ -1203,6 +1203,30 @@ tracepoint(CLOG_DATAPATH_WINUSER_C, DatapathErrorStatus , arg2, arg3, arg4);\
 
 
 
+#ifndef _clog_5_ARGS_TRACE_DatapathErrorStatus
```

Comment:
I really don't want to know what this is, but FYI it looks suspect because you've added QuicTraceEvent calls with two different strings but there's only one... thing... added here.

Reply:
Yeah, the generated code here is REALLY weird. It generates just the first log for each type in the file and just reuses that. Since the last string is passed in as a string and not part of the log.

Reply:
OK cool, just so long as it isn't broken!